interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md

```diff
@@ -4,15 +4,15 @@ owner: Mary Sammal
 ---
 # Introduction to Banded Reports
 
-Banded reports provide a generalized report layout notion. When you preview a banded report, a report document is generated based on the report layout you designed and the data source you specified.
+Banded reports provide a generalized report layout notion. A report document is generated based on the specified data source and the report layout you designed.
```

Comment:
I cannot remove the "When you preview a banded report," part. At the time a reader reads this topic, he/she does not know that a report is what they design and a report document is generated based on the report layout they designed and the data source they specified, when they preview the report. If we remove this part, "A report document" that starts the next sentence is not clear for the reader.